src/assert.js

```diff
@@ -86,8 +86,17 @@ class Assert {
 		const resume = internalStop( test );
 
 		return function done() {
+
+			if ( config.current === undefined ) {
+				throw new Error( "`assert.async` callback from test '" +
```

Comment:
Very slight preference for an escaped double quote here to match the formatting of errors from `assert.rejects`.

I might explore doing a larger consistency clean up over these kinds of messages. One of the things I quite liked from another patch recently is placing the tes name on its own line separate from the error. That seems to help both in brevity (problem first, then context), as well as less likely to become difficult to read when test names contain quotes and other symbols, e.g. `assert.async() callback after test finished.\nTest: foo bar's "baz"`. But, that's for another one.

I'll swap the quote here when landing the PR, no amend needed just for that.